09-soil_exploration_module.Rmd

```diff
@@ -182,142 +176,162 @@ Great! Now let's calculate some basic statistics. For example, we might want to
 :::
 
-Let's say we're interested in looking at mean concentrations that were determined using EPA Method 3051. Given that there are 8 of these measures in the `soil.values` dataset, it would be time consuming to run our code from above for each individual measure.
+Let's say we're interested in looking at mean concentration for any element that was determined using EPA Method 3051. Given that there are 8 of these measures in the `soil.combined` dataset, it would be time consuming to run our code from above that calculates overall mean for each individual measure.
 
 We can add two arguments to our `summarize` statement to calculate statistical measures for multiple columns at once: the `across` argument, which tells R to apply the `summarize` command to multiple columns; and the `ends_with` parameter, which tells R which columns should be included in the statistical calculation.
 
-We are using `ends_with` because for this question, all the columns that we're interested in end with the string 'EPA3051'. 
+We are using `ends_with` because for this question, all the columns that we're interested in end with the string 'EPA3051'.
 
 This command follows the code structure:
 
-dataset %>%
-    group_by(column_name) %>%
-    summarize(across(ends_with(common_column_name_ending), mean))
+dataset %>% group_by(column_name) %>% summarize(across(ends_with(common_column_name_ending), mean))
 
 ```{r}
-soil.values.clean %>%
-    group_by(region) %>%
+soil.combined %>%
+    group_by(origin) %>%
     summarize(across(ends_with('EPA3051'), mean))
-
-
 ```
 
-This is a _much_ more efficient way to calculate statistics.
+This is a *much* more efficient way to calculate statistics.
 
-::: {.reflection}
+::: reflection
 QUESTIONS:
 
-8. Calculate the maximum values for concentrations that were determined using EPA Method 3051. (HINT: change the function you call in the `summarize` statement.) Which of these metals has the maximum concentration you see, and in which region is it found?
-
-9. Calculate both the mean and maximum values for concentrations that were determined using the Mehlich3 test. (HINT: change the terms in the `columns_to_include` vector, as well as the function you call in the `summarize` statement.) Which of these metals has the highest average and maximum concentrations, and in which region are they found?
+8.  Calculate the maximum values for concentrations that were determined using EPA Method 3051. (HINT: change the function you call in the `summarize` statement.) Which of these metals has the maximum concentration you see, and in which region is it found?
 
+9.  Calculate both the mean and maximum values for concentrations that were determined using the Mehlich3 test. (HINT: change the terms in the `columns_to_include` vector, as well as the function you call in the `summarize` statement.) Which of these metals has the highest average and maximum concentrations, and in which region are they found?
```

Comment:
when did you make the `columns_to_include` vector? I'm not following

Reply:
I didn't - it's poorly written. I have fixed.